crates/telio-model/src/api_config.rs

```diff
@@ -219,12 +219,16 @@ pub struct Features {
     pub direct: Option<FeatureDirect>,
     /// Test environment (natlab) requires binding feature disabled
     /// TODO: Remove it once mac integration tests support the binding mechanism
-    #[cfg(any(target_os = "macos", feature = "pretend_to_be_macos"))]
```

Comment:
UniFFI doesn't like conditionally including/excluding stuff it exposes so this field will now always be there